db/migrations/migrations_test.go

```diff
@@ -98,6 +98,57 @@ func (s *MigrationTestSuite) TestBCDAMigration() {
 				}
 			},
 		},
+		{
+			"Add type column to cclf_files",
+			func(t *testing.T) {
+				migrator.runMigration(t, "2")
+				noType := &models.CCLFFile{
```

Comment:
Should we also support a test case where we explicitly define `FileTypeDefault`?

Reply:
> Should we also support a test case where we explicitly define `FileTypeDefault`?

The `noType` here should verify the `FileTypeDefault` case (since the zero value for CCLFFile is `FileTypeDefault`). But I can add an explicit one if you think that behavior should be called out.

Reply:
I was thinking an explicit test case would be a good idea (3 tests in total), but I'm ok if we just use the `noType` case.